docs/JAX_visualizing_models_metrics.md

```diff
@@ -0,0 +1,252 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.15.2
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# JAX and Tensorboard / NNX Display
+
++++
+
+To keep things straightforward and familiar, we reuse the model and data from the 'getting_started_with_jax_for_AI' notebook - if you haven't read that yet and want the primer, start there before returning.
```

Comment:
We should link to https://jax-ai-stack.readthedocs.io/en/latest/getting_started_with_jax_for_AI.html and remove the github source tree links below.

Reply:
Thanks for the catch - updated!